tests/integration/local/start_api/lambda_authorizers/test_start_api_lambda_authorizers.py

```diff
@@ -0,0 +1,259 @@
+        ),
+        (
+            {"AuthSimpleHandler": "app.unauthv2", "AuthOverride": "RequestAuthorizerV2Simple"},
+            "/testdata/start_api/lambda_authorizers/serverless-http-props.yaml",
+        ),
+        (
+            {"AuthHandler": "app.unauth", "AuthOverride": "Token"},
+            "/testdata/start_api/lambda_authorizers/serverless-api-props.yaml",
+        ),
+        (
+            {"AuthHandler": "app.unauth", "AuthOverride": "Request"},
+            "/testdata/start_api/lambda_authorizers/serverless-api-props.yaml",
+        ),
+    ],
+)
+class TestServerlessPropertiesLambdaAuthorizersUnauthorized(StartApiIntegBaseClass):
+    def setUp(self):
+        self.url = f"http://127.0.0.1:{self.port}"
+
+    @pytest.mark.flaky(reruns=3)
+    @pytest.mark.timeout(timeout=600, method="thread")
+    def test_unauthorized_request(self):
+        headers = {"header": "myheader"}
+        query = {"query": "myquery"}
+
+        response = requests.get(f"{self.url}/requestauthorizer", headers=headers, params=query, timeout=300)
+        response_json = response.json()
+
+        self.assertEqual(response.status_code, 403)
+        self.assertEqual(response_json, {"message": "User is not authorized to access this resource"})
+
+
+@parameterized_class(
+    ("parameter_overrides", "template_path"),
+    [
+        (
+            {"AuthHandler": "app.throws_exception", "AuthOverride": "RequestAuthorizerV1"},
+            "/testdata/start_api/lambda_authorizers/serverless-http-props.yaml",
+        ),
+        (
+            {"AuthSimpleHandler": "app.throws_exception", "AuthOverride": "RequestAuthorizerV2Simple"},
+            "/testdata/start_api/lambda_authorizers/serverless-http-props.yaml",
+        ),
+        (
+            {"AuthHandler": "app.throws_exception", "AuthOverride": "Token"},
+            "/testdata/start_api/lambda_authorizers/serverless-api-props.yaml",
+        ),
+        (
+            {"AuthHandler": "app.throws_exception", "AuthOverride": "Request"},
+            "/testdata/start_api/lambda_authorizers/serverless-api-props.yaml",
+        ),
+    ],
+)
+class TestServerlessPropertiesLambdaAuthorizer500(StartApiIntegBaseClass):
+    def setUp(self):
+        self.url = f"http://127.0.0.1:{self.port}"
+
+    @pytest.mark.flaky(reruns=3)
+    @pytest.mark.timeout(timeout=600, method="thread")
+    def test_unauthorized_request(self):
+        headers = {"header": "myheader"}
+        query = {"query": "myquery"}
+
+        response = requests.get(f"{self.url}/requestauthorizer", headers=headers, params=query, timeout=300)
+        response_json = response.json()
+
+        self.assertEqual(response.status_code, 502)
+        self.assertEqual(response_json, {"message": "Internal server error"})
+
+
+class TestUsingSimpleResponseWithV1HttpApi(WritableStartApiIntegBaseClass):
+    do_collect_cmd_init_output = True
+    template_content = """AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Yeah, the setup stuff is abstracted away in the parent class, hence why there is only the template definition here instead of the code to write to a file. I can move this logic to a new base class specifically designed to template validation failures for `start-api` if that sounds alright to you. This will also resolve comment above this one about the general catch all `Error: ` I added.

Reply:
The test case should define what is going on to make it easier for us to know what is covering, but any way it is not a big deal, we can just add some comments to make it easier for people to read what is happening

Reply:
I can change the class name to make it more clear that we are expecting a failure and to add a comment pointing out where we are introducing the failure at.